changelog/ob.md

```diff
@@ -100,34 +102,18 @@ until after the last use of the borrowed pointer.
 A Borrowed pointer must have the `scope` attribute and must
 be a pointer to mutable.
 
-4. Lent
-
-A Lent pointer loaned out its value to a Borrowed pointer. While
-in the Lent state the pointer cannot have its value read or written
-to.
-A Lent pointer must not be `scope` and must be a pointer to mutable.
-
-5. Readonly
+4. Readonly
 
 A Readonly pointer acquires its value from an Owner. Afterwards,
 the Owner enters the View state.
```

Comment:
What is a 'View' pointer ? :P

Reply:
A vestige of the previous version. Removed.